examples/twittermap/web/public/javascripts/common/services.js

```diff
@@ -40,13 +39,14 @@ angular.module('cloudberry.common', [])
       }
     };
   })
-  .service('cloudberry', function($http, $timeout, $location, cloudberryConfig) {
+  .service('cloudberry', function($http, $timeout, $location, cloudberryConfig, ResultCache) {
     var startDate = config.startDate;
     var endDate = config.endDate;
     var defaultNonSamplingDayRange = 1500;
     var defaultSamplingDayRange = 1;
     var defaultSamplingSize = 10;
     var ws = new WebSocket(cloudberryConfig.ws);
+    var geoIdsNotInCache = [];
```

Comment:
Add a comment for this variable? What is it? Where is it used?

Reply:
Required comments have been added for clarity.